pom.xml

```diff
@@ -84,8 +75,34 @@
 		<module>de.tudarmstadt.ukp.dkpro.argumentation.preprocessing</module>
 		<module>de.tudarmstadt.ukp.dkpro.argumentation.misc</module>
 		<module>de.tudarmstadt.ukp.dkpro.argumentation.examples</module>
+		<module>de.tudarmstadt.ukp.dkpro.argumentation.fastutil</module>
+		<module>de.tudarmstadt.ukp.dkpro.argumentation.nio</module>
 	</modules>
 
+	<build>
```

Comment:
Me too, but (at least right now) the only sub-project which absolutely *requires* Java 8 is `de.tudarmstadt.ukp.dkpro.argumentation.nio`. Assuming that this fork gets merged (see [previous comment on merging/separating the code](https://github.com/dkpro/dkpro-argumentation/pull/5#discussion_r68024518)), this can then be handled at the parent-POM level (thus building all sub-projects to conform to Java 8 syntax/standard libraries).